quick-start/src/test/java/com/marklogic/quickstart/integrationtests/EndToEndAPITest.java

```diff
@@ -0,0 +1,350 @@
+package com.marklogic.quickstart.integrationtests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.hub.HubTestBase;
+import com.marklogic.hub.flow.CodeFormat;
+import com.marklogic.hub.flow.DataFormat;
+import com.marklogic.hub.flow.FlowType;
+import com.marklogic.quickstart.model.entity_services.EntityModel;
+
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class EndToEndAPITest extends HubTestBase {
+
+	@LocalServerPort
+	int port;
+
+	private IntegrationUtils utils = new IntegrationUtils();
+	
+	// Project Initialization variables
+	boolean projectInitStatus;
+	JsonPath projectInitJson;
+	Response projectInitResponse;
+	String projectPath;
+	
+	// Login Process variables
+	String loginResponse;
+	
+	// Clear Database variables
+	Response clearDbResponse;
+	
+	// Database stats variables
+	JsonPath statsJson;
+	Response statsResponse;
+	
+	// Create Entities variables
+	JSONObject createEntityJsonObj;
+	JsonPath getEntitiesJson;
+	List<EntityModel> entitiesList;
+	Response createEntityResponse;
+	Response getEnitiesResponse;
+	String createEntityJsonBody;
+	String entityName;
+	String responseEntityTitle;
+	
+	// Flow process variables
+	JsonPath createInputFlowJson;
+	JsonPath createHarmonizeFlowJSON;
+	JsonNode mlcpOptions;
+	JsonNode runHmJsonObj;
+	Map<String, Object> runFlowOptions;
+	Response createInputFlowResponse;
+	Response runIpFlowResponse;
```

Comment:
try to be consistent using abbreviations or not.  best to keep the full names in Java.

Reply:
Updated to consistent variable names. Using full names instead of abbreviations.